@here/harp-materials/lib/MapMeshMaterials.ts

```diff
@@ -715,6 +728,41 @@ export class FadingFeatureMixin implements FadingFeature {
     }
 }
 
+/**
+ * Class to apply the code necessary to customize the standard shader to remove the diffuse light
+ * component.
+ */
+export class ShadowFeatureMixin {
```

Comment:
While current mixins live in kitchen sink `MapMeshMaterials` i would propose to move all stuff related to ShadowFeature to separate file ... (like `ShadowFeature.ts` ?)

Reply:
Agreed, this file is very hard to move about, I think however that all parts should be separated (i.e. the mixins separated from the materials).